conn-fix/README.md

```diff
@@ -367,5 +367,8 @@ defaultHeartbeatInterval: 5
 
 ## Release notes
 
+### 3.4.2
++ Fixed the problem conn publishes raw message incoming from `send` pin with short header (8, 9, 35 tags only) (generated by codec)
```

Comment:
```suggestion
+ Fixed the problem in which conn publishes incoming raw message from `send` pin with short header (8, 9, 35 tags only) (generated by codec)
```

Reply:
@jean-michel-perli-sonsin we have the workflow for conn :
1. receives a raw message **from the pin with attribute `send`** with a **short header**
2. **enriches the header** before sending the message to the remote system
3. sends the message to a remote system
4. sends the same message to th2

source problem is conn sent the message with source contend from step 1 instead of enriched version

Reply:
@Nikita-Smirnov-Exactpro thank you for the clarification. Maybe in this case it would be clear if we say: 
- Fixed the problem in which conn sends the message to the remote system with short header instead of enriched version

Reply:
Sorry, it is not correct, because it sends the correct message to remote system, but it sends an incorrect message into th2

Reply:
@jean-michel-perli-sonsin could you please consider the sentence
Fixed the problem in which conn sends a correct user's message to the remote system but sends an incorrect copy of it to the th2 with short header instead of an enriched version

Reply:
@Nikita-Smirnov-Exactpro I agree with the sentence that you proposed, it’s clear now for me and eventually for the users